packages/atomic/src/components/atomic-folded-result-list/atomic-folded-result-list.stories.tsx

```diff
@@ -0,0 +1,402 @@
+import {html} from 'lit-html';
+import defaultStory from '../../../.storybook/default-story';
+
+const {defaultModuleExport, exportedStory} = defaultStory(
+  'Atomic/FoldedResultList',
+  'atomic-folded-result-list',
+  {},
+  {
+    engineConfig: {
+      search: {
+        preprocessSearchResponseMiddleware: (response) => {
+          response.body.results[0] = resultWithChildren;
+          return response;
+        },
+      },
+    },
+    additionalChildMarkup: () => html`
+      <atomic-result-template>
+        <template>
+          <atomic-result-link></atomic-result-link>
+          <div style="padding-left: 15px; margin-top: 20px;">
+            <atomic-result-children>
+              <b slot="before-children">Replies:</b>
+              <atomic-result-children-template>
+                <template>
+                  <atomic-result-link></atomic-result-link>
+                </template>
+              </atomic-result-children-template>
+            </atomic-result-children>
+          </div>
+        </template>
+      </atomic-result-template>
+    `,
+  }
+);
+
+const resultWithChildren = {
```

Comment:
I think what Benjamin indexed recently is a more appropriate content

Reply:
Could be accessed with an `aq` instead of mocking the response

Reply:
took me my time to find out what `aq` meant, but seems like a much better approach, thanks!